Code/GraphMol/ForceFieldHelpers/CrystalFF/TorsionPreferences.cpp

```diff
@@ -220,7 +232,7 @@ void getExperimentalTorsions(const RDKit::ROMol &mol, CrystalFFDetails &details,
         // FIX: check if bond is NULL
         bid2 = mol.getBondBetweenAtoms(aid2, aid3)->getIdx();
         // check that a bond is part of maximum one ring
-        if (mol.getRingInfo()->numBondRings(bid2) > 1 ||
+        if (mol.getRingInfo()->numBondRings(bid2) > 3 ||
```

Comment:
Not sure I understand this change. Is it intentional? If so, the comment in the line above should probably also be updated.